static/js/flatpickr.js

```diff
@@ -39,6 +39,10 @@ export function show_flatpickr(element, callback, default_timestamp, options = {
 
             const hotkey = get_keydown_hotkey(event);
 
+            if (!hotkey) {
+                return;
+            }
```

Comment:
The block previously fell through to do `event.stopPropagation` for keys that include tab/shift_tab; is that change in behavior intentional?

I'm actually a bit confused -- maybe we should add a block comment at the top of this handler explaining what it's responsibilities are -- might be able to mine that information from the commit history.

Reply:
> The block previously fell through to do `event.stopPropagation` for keys that include tab/shift_tab;

We [still call `event.stopPropagation`](https://github.com/zulip/zulip/pull/23356/files#diff-92bdae0e2a79ce4cc5b6dba6265af6831efac7db3969fe2628e0f2de8bd4bdeaR65) if the key includes `tab` or `shift_tab`.

> I'm actually a bit confused -- maybe we should add a block comment at the top of this handler explaining what it's responsibilities are

Yeah, that's a good idea, check if [this](https://github.com/zulip/zulip/pull/23356/files#diff-92bdae0e2a79ce4cc5b6dba6265af6831efac7db3969fe2628e0f2de8bd4bdeaR33-R39) clarify the doubts about `onKeyDown` handler.